pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -38,6 +38,40 @@
       "InboxOn": {}
     },
     "Microsoft.Build": {
+      "StableVersions": [
```

Comment:
Technically we don't care about StableVersions for packages we don't build nor depend on in CoreFx.  These are present in the index because they were part of the desktop targeting pack.  Not sure we care enough to remove these, but it is an unnecessary diff that will grow over time.

Reply:
Right, I'll remove all of these